traffic_ops/traffic_ops_golang/riak.go

```diff
@@ -15,25 +34,177 @@ import (
 )
 
 const RiakPort = 8087
-const CdnUriKeysBucket = "cdn_uri_sig_keys" // riak namespace for cdn uri signing keys.
+const CdnURIKeysBucket = "cdn_uri_sig_keys" // riak namespace for cdn uri signing keys.
```

Comment:
sorry missed this one CdnURIsKeyBucket should be CDNURIKeysBucket (CDN is an acronym)